apis/s3/v1alpha3/bucketpolicy_types.go

```diff
@@ -53,6 +52,22 @@ type BucketPolicyParameters struct {
 	BucketNameSelector *xpv1.Selector `json:"bucketNameSelector,omitempty"`
 }
 
+// BucketPolicyBody represents an S3 bucket policy in the manifest
+type BucketPolicyBody struct {
+	// Version is the current IAM policy version
+	Version string `json:"version"`
```

Comment:
Should this be an enum for either `2012-10-17` or `2008-10-17`. Those are the only two valid options I see from this IAM policy doc https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_version.html (not sure if s3 policy is different). 

Secondary question, should default to `2012-10-17`, unless this is being late initialized with AWS default?